target/scripts/helpers/log.sh

```diff
@@ -1,11 +1,28 @@
 #! /bin/bash
 
-LOG_RESET='\e[0m'
-LOG_LGRAY='\e[37m'
-LOG_LBLUE='\e[94m'
-LOG_BLUE='\e[34m'
-LOG_LYELLOW='\e[93m'
-LOG_RED='\e[91m'
+# shellcheck disable=SC2291 # Quote repeated spaces to avoid them collapsing into one.
+# shellcheck disable=SC2034 # VAR appears unused.
+
+# Color variables for global usage
+RED=$(echo -ne     '\e[0;31m')
+GREEN=$(echo -ne   '\e[0;32m')
+YELLOW=$(echo -ne  '\e[0;33m')
+BLUE=$(echo -ne    '\e[0;34m')
+PURPLE=$(echo -ne  '\e[0;35m')
+CYAN=$(echo -ne    '\e[0;36m')
+WHITE=$(echo -ne   '\e[0;37m')
+
+# Light/bold variants
+LRED=$(echo -ne    '\e[1;31m')
+LGREEN=$(echo -ne  '\e[1;32m')
+LYELLOW=$(echo -ne '\e[1;33m')
+LBLUE=$(echo -ne   '\e[1;34m')
+LPURPLE=$(echo -ne '\e[1;35m')
+LCYAN=$(echo -ne   '\e[1;36m')
+LWHITE=$(echo -ne  '\e[1;37m')
+
+ORANGE=$(echo -ne  '\e[38;5;214m')
+RESET=$(echo -ne   '\e[0m')
```

Comment:
I was just thinking: maybe we should prefix this with `COLOR_`?

Reply:
I thought about the same. But then decided to omit it:

- i think it's clear from the variable name what the usage is
- better readability in scripts with lots of colors (IMO), e.g. `${COLOR_LYELLOW}` is quiet long compared to `${YELLOW}`.

But I don't mind. We can change this. Lets wait what other maintainers say.

Reply:
I'm absolutely fine with it as is, just wanted to ask here before we merge this and think about it later :)